modules/manifest.js

```diff
@@ -40,8 +41,15 @@ class Manifest {
     this.status = options.status;
     this.user = options.user;
     this.images = options.images;
-    this.labels = options.labels;
-    this.labelPos = options.labelPos;
+
+    this.imOptions = {};
+    this.imOptions.equalize = options.equalize;
+
+    if( exists(options.labels) ) {
```

Comment:
I don't think we should need this helper when checking for whether a property exists on an object:

```javascript
$ node
> var a = {}
> a
{}
> console.log('a.b', a.b ? 'exists' : 'does not exist')
a.b does not exist
> a.b = 'c'
'c'
> console.log('a.b', a.b ? 'exists' : 'does not exist')
a.b exists
```